python/shotgun_model/shotgun_hierarchy_model.py

```diff
@@ -961,16 +938,22 @@ def hierarchy_is_supported(sg_connection):
 
     :param sg_connection: A shotgun connection.
 
-    :returns: ``True`` if hierarchy supported, ``False`` otherwise.
+    :returns: A tuple of 2 items where the first item is a boolean indicating
```

Comment:
I've been trying to use `:param type name:` as well when possible. I haven't been using `:rtype:` as much, but I like it. Updated here.